src/Compilers/CSharp/Portable/Symbols/Metadata/PE/NativeIntegerTypeDecoder.cs

```diff
@@ -31,11 +31,24 @@ private static TypeSymbol TransformType(TypeSymbol type, ImmutableArray<bool> tr
                 {
                     return result;
                 }
+                else
+                {
+                    return new UnsupportedMetadataTypeSymbol();
```

Comment:
>return new UnsupportedMetadataTypeSymbol(); [](start = 20, length = 43)

We shouldn't be doing this. We should simply ignore malformed attribute as we did before